scripts/travis-reprotest

```diff
@@ -20,39 +22,29 @@ keep_alive_pid=$!
 
 trap 'exit_travis travis-reprotest.log ${keep_alive_pid}' 0 1
 
-# we handle case where we run reprotest stage separately with build artifacts (e.g. GitlabCI).
-if [ "$1" = prepare-chroot ]; then
-    prepare_chroot
+YUM="$(/usr/bin/which dnf || true)"
+if [ -z "$YUM" ]; then
+    YUM="$(/usr/bin/which yum || true)"
 fi
-
-chroot_dir="$(ls -d chroot-* || :)"
-if [ -z "$chroot_dir" ]; then
-    exit
-fi
-
-YUM="$(get_yum "$chroot_dir")"
-APT="$(get_apt "$chroot_dir")"
+APT="$(/usr/bin/which apt-get || true)"
 
 if [ "$APT" ]; then
-    APT_ENV="DEBIAN_FRONTEND=noninteractive DEBCONF_NOWARNINGS=yes DEBIAN_PRIORITY=critical"
-    APT_OPTS="-o Dpkg::Options::=--force-confnew --yes"
+    # copy qubes-builder packages repo to /tmp/qubes-repo
+    builder_repo_dir="$builderdir/qubes-packages-mirror-repo/$([[ -n ${DIST_DOM0} ]] && echo "dom0-${DIST_DOM0}" || echo "vm-${DISTS_VM}")"
+    sudo mkdir -p /tmp/qubes-deb
+    sudo mount --bind "$builder_repo_dir" /tmp/qubes-deb
+
+    apt-get update
```

Comment:
Why does it work? Is the whole job in docker running as root?

Reply:
Ah yes in this docker image I've not setup `gitlab-runner` user like in https://github.com/fepitre/qubes-g2g-continuous-integration/blob/bd912baaa7e1c10594da9c884337a0727c297798/runner/docker/qubesos-ci/Dockerfile#L13. In both cases it needs a fix with at least sudo call.